src/arbiter.rs

```diff
@@ -5,11 +5,12 @@ use std::thread;
 use futures::sync::oneshot::{channel, Sender};
 use futures::{future, Future, IntoFuture};
 use tokio::executor::current_thread::spawn;
-use tokio::runtime::current_thread::Runtime;
+use tokio::runtime::current_thread::Builder as RuntimeBuilder;
 use uuid::Uuid;
 
 use actor::Actor;
 use address::{channel, Addr, AddressReceiver};
+use clock::Clock;
```

Comment:
Do you think we should re-export this time for user convenience?
Since we require to pass its instance in API?

Reply:
You mean from the arbiter module?
It is re-exported [here](https://github.com/actix/actix/pull/137/files/ec20065dd024eac283e242f6c641e6ff107dea2f#diff-438dff5ffb2f02c4553ee7c50728f6e5R11). I'm not sure if this is "important" enough to go in the prelude, though...

Reply:
Ah yes, sorry, just briefly looked it over since it wasn't supposed to change.
if it is publicly available to user then I have no problems :)